src/renderer/screens/Editor/Macros/MacroEditor.js

```diff
@@ -0,0 +1,156 @@
+// -*- mode: js-jsx -*-
+/* Chrysalis -- Kaleidoscope Command Center
+ * Copyright (C) 2020-2022  Keyboardio, Inc.
+ *
+ * This program is free software: you can redistribute it and/or modify it under
+ * the terms of the GNU General Public License as published by the Free Software
+ * Foundation, version 3.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+import React, { useEffect, useState } from "react";
+import { useTranslation } from "react-i18next";
+
+import Focus from "@api/focus";
+import { KeymapDB } from "@api/keymap";
+
+import Box from "@mui/material/Box";
+import Card from "@mui/material/Card";
+import CardContent from "@mui/material/CardContent";
+import CardHeader from "@mui/material/CardHeader";
+import IconButton from "@mui/material/IconButton";
+import Stack from "@mui/material/Stack";
+
+import CloseIcon from "@mui/icons-material/Close";
+
+import MacroStep from "./MacroStep";
+import MacroStepAdd from "./MacroStepAdd";
+import MacroStepEditor from "./MacroStepEditor";
+
+const focus = new Focus();
```

Comment:
Do we use this?

Reply:
We do not, good catch. Thanks! We don't use KeymapDB + db, either.